jobs/release.Jenkinsfile

```diff
@@ -138,14 +138,26 @@ lock(resource: "release-${params.STREAM}", extra: locks) {
             }
 
 
-            if ((basearch in ['aarch64', 'x86_64']) && params.AWS_REPLICATION) {
+            if (params.AWS_REPLICATION &&
+                shwrapCapture("cosa meta --arch=${basearch} --get-value amis") != "None") {
                 // Replicate AMI to other regions.
-                stage("Replicate ${basearch} AWS AMI") {
-                    withCredentials([file(variable: 'AWS_CONFIG_FILE',
-                                          credentialsId: 'aws-build-upload-config')]) {
-                        shwrap("""
-                        cosa aws-replicate --build=${params.VERSION} --arch=${basearch} --log-level=INFO
-                        """)
+                stage("${basearch} AWS AMI Replication") {
+                    def replicated = false
+                    def credentials = [file(variable: 'AWS_CONFIG_FILE',
```

Comment:
Minor: I guess now we could reflow as

```groovy
def ids = ['aws-build-upload-config', 'aws-govcloud-image-upload-config']
for (id in ids) {
	tryWithCredential([file(variable: 'AWS_CONFIG_FILE', credentialsId: id)]) {
		...
	}
}
```

to avoid repeating some of the boilerplate stuff.